### PR TITLE
Doc - UG Updates caa 08042022 2138H

### PR DESCRIPTION
**Pull request by lchokhoe**
# Doc - UG Updates caa 08042022 2138H

## Key Summary
- The implemented way to handle removal of any Tags using `edit` command has been included in the `edit` subsection of the `Features` section as an `Advanced Function`.

## Other Things to Note
- The minimum phone number of 3 digits is now mentioned in the `Notes about the input format` subsection of the `Features` section.
- INDEX for `delete` and `edit` command must be natural numbers and must not exceed 2147483647. This is now mention in the respective subsections.
- "How many applications can SIA track?" added to FAQ with an answer.
- "When I use `find` followed by `edit` sometimes the application that I edited disappears form the main window. Am I doing something wrong?" added to FAQ with an answer.

## Feedback that I would like
Please take a look at the wording for the removal of and Tags using `edit`. I would like some feedback on whether it makes sense. Again, if you have the time please help me with trying out the feature on your local device.

Ready for review @joszx @laughingkid-sg @yongler @tanyutao544 (anyone will do)

Addresses issue #164 and #179

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -99,9 +99,10 @@ Edit at your own risk of losing data.
 
 
 **:information_source: Notes about the input format:**<br>
+* For `[p/Phone]`, a minimum of 3 digits must be inputted but most phone numbers would be at least 8 digits long.
 
-* For `[t/TAG]...`, only alphanumeric inputs are allowed. i.e. Only the characters A-Z, a-z, 0-9.<br>
-  e.g. `t/Based In Singapore` is not allowed, `t/BasedInSingapore` is allowed.
+* For `[t/TAG]...`, only alphanumeric inputs are allowed and cannot be empty. i.e. Only the characters A-Z, a-z, 0-9.<br>
+  e.g. `t/Based In Singapore` or `t/` is not allowed, `t/BasedInSingapore` is allowed.
```
Comment (tanyutao544) [inline docs/UserGuide.md]: We might need to add a note that they cannot put t/High or t/applied

File: docs/UserGuide.md [hunk 2]
```diff
@@ -196,32 +198,55 @@ Edits an existing application in SoC InternApply.
...
+- If removing any of the Tags from an application causes it to become a duplicate application the command will be invalid. Please refer to our Notes about duplicate applications
+
+**Example usages and expected outcomes**
+
+- `edit 1 t/removetags t/removepriority` Removes all the Optional Tags and the Priority Tag from the 1st application.
+- `edit 1 p/65258719 t/removestatus` Removes the Application Status Tag from the 1st application. Edits the phone number of the 1st application to be `65258719`
+- `edit 1 pt/HIGH t/removepriority` Special Tags takes precedence over all other Tag inputs. Removes the Priority Tag from the 1st application. Ignores the `pt/HIGH` input.
+- `edit 1 t/removestatus t/Singapore t/ExEmployee` Special Tags takes precedence over all other Tag inputs. Removes the Application Status Tag from the 1st application. Ignores the `t/Singapore t/ExEmployee` inputs.
+- `edit 1 t/removestatus` followed by `edit 1 t/removestatus` First removes the Application Status Tag from the 1st application. Then does nothing since there does not exist an Application Status Tag for the 1st application.
```
Comment (tanyutao544) [inline docs/UserGuide.md:252]: LGTM, smart separating more complicated functions into a different section, might want to consider separating edit function here too

**Review by tanyutao544 — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (lchokhoe) [reply docs/UserGuide.md]: Good shout! Made an additional note about this in the same subsection. Also reordered the points so that it reads better.

**Pull request merged**